### PR TITLE
Extract translation files

### PR DESCRIPTION
**Pull request by quba42**
auto generated using the following command from within forklift:

bundle exec rake plugin:gettext[foreman_scc_manager]

### CODE REVIEW

**Comment by quba42** [comment]
Excluded the `locale` directory from rubocop, since it contains auto generated files.

File: locale/de/foreman_scc_manager.po [hunk 1]
```diff
@@ -123,7 +125,7 @@ msgstr ""
 msgid "Incremental Update"
 msgstr ""
 
-msgid "Incremental Update of 0 Content View Version(s) "
+msgid "Incremental Update of  Content View Version(s) "
```
Comment (adamruzicka) [inline locale/de/foreman_scc_manager.po]: there should be a placeholder for the number of CVVs

File: locale/action_names.rb [hunk 2]
```diff
@@ -1,61 +1,73 @@
...
-_('Disable')
-_('Sync SUSE subscriptions (Repositories)')
-_('Subscribe SCC Product')
-_('Remove Versions and Associations')
-_('Synchronize')
-_('Publish')
-_('Update Sync Plan Products')
-_('Promotion')
-_('Update Sync Plan')
+_("Incremental Update")
```
Comment (quba42) [inline locale/action_names.rb]: I am commenting on this line as an example.
There appear to be a lot of strings in this automatically generated file, that don't appear anywhere in the plugin. (I think they are from Katello).

`"Incremental Update"` is just one example.
By comparing with strings found using `git grep '_('` it looks like about half these strings should not be here.

**Comment by m-bucher** [comment]
So for the current state I more or less manually removed all strings that seemed unnecessary to me.
Not really the most efficient way of doing this :unamused:

File: locale/de/foreman_scc_manager.po (hunk 1 above)
Comment (quba42) [reply locale/de/foreman_scc_manager.po]: String was manually removed since it is from Katello.

File: locale/action_names.rb (hunk 2 above)
Comment (quba42) [reply locale/action_names.rb]: Solved by manually removing the offending strings.

**Pull request merged**